src/main/java/seedu/address/logic/parser/FilterByDateCommandParser.java

```diff
@@ -0,0 +1,46 @@
+package seedu.address.logic.parser;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.Optional;
+
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.ListCommand;
+import seedu.address.model.filter.FilterByDate;
+import seedu.address.model.filter.TaskPredicate;
+
+public class FilterByDateCommandParser implements Parser<Command>{
+
```

Comment:
That's not how our `Command` works, the `CommandParser.parse()` parse a command line and then return a new `Command` object.